project/src/main/world/creature/Body2Colors.tscn

```diff
@@ -0,0 +1,67 @@
+[gd_scene load_steps=4 format=2]
+
+[ext_resource path="res://src/main/world/creature/creature-curve.gd" type="Script" id=1]
+[ext_resource path="res://src/main/world/creature/body-colors.gd" type="Script" id=2]
+
+[sub_resource type="Curve2D" id=1]
+resource_local_to_scene = true
+_data = {
+"points": PoolVector2Array( -127.143, 30.6224, 168.544, -40.5937, 351.338, -9.41907, 40.1355, 128.205, -31.4327, -100.406, 487.725, -300.49, 42.9136, 5.92072, -43.5736, -6.01178, 382, -484, 0, 0, 0, 0, 364, -386, 45.8762, 3.59069, -42.6172, -3.33561, 257.593, -536.035, 0, 0, 0, 0, 236, -422, 47.9093, -20.2282, -39.8701, 16.8339, 125.168, -510.071, 0, 0, 0, 0, 120, -346, 34.7119, -10.354, -34.0173, 10.1468, 33.7709, -421.671, -15.991, -99.3122, 19.1425, 118.885, -12.5286, -228.756, -226.018, 33.3689, 125.149, -18.4767, 344.901, -9.12219 )
+}
+
+[node name="Body" type="Node2D"]
+position = Vector2( 580, 850 )
+script = ExtResource( 2 )
+creature_visuals_path = NodePath("../../..")
```

Comment:
Unset these CreatureVisualsPaths; they'll be set when they're added.

Reply:
Unset creature_visuals_path in FatSpriteMover1.tscn, FatSpriteMover2.tscn, Body1.tscn, Body2.tscn, Body2Colors.tscn.